client/app/veeam/dashboard/veeam-dashboard.html

```diff
@@ -1,58 +1,123 @@
+                data-loading="$ctrl.subscriptionInfos.loading"
+                class="oui-tile-h-100 h-100">
+                <oui-tile-definition data-ng-if="$ctrl.subscriptionInfos.loading">
+                    <oui-tile-description>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <div data-ng-if="!$ctrl.subscriptionInfos.loading">
+                    <oui-tile-definition
+                        data-ng-if="$ctrl.subscriptionInfos.data.offer" 
+                        data-term="{{ ::'common_tile_subscription_offer' | translate }}" data-description="{{ ::$ctrl.subscriptionInfos.data.offer }}">
+                        <oui-action-menu data-compact data-align="end" data-ng-if="$ctrl.uiActions.changeOffer.isAvailable()">
+                            <oui-action-menu-item data-text="{{::$ctrl.uiActions.changeOffer.text}}"
+                                data-aria-label="{{::$ctrl.uiActions.changeOffer.text}}"
+                                data-on-click="$ctrl.uiActions.changeOffer.callback()">
+                            </oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{($ctrl.subscriptionInfos.data.renew.forced || $ctrl.subscriptionInfos.data.renew.automatic) ? 'common_tile_subscription_autorenew' : 'common_tile_subscription_expiration_date' | translate}}"
+                        data-description="{{ ::$ctrl.subscriptionInfos.data.expiration | momentFormat:'LL' }}">
+                        <oui-action-menu data-compact data-align="end" data-ng-if="$ctrl.uiActions.manageAutorenew.isAvailable()">
+                            <oui-action-menu-item data-text="{{ ::$ctrl.uiActions.manageAutorenew.text }}"
+                                data-aria-label="{{ ::$ctrl.uiActions.manageAutorenew.text }}"
+                                data-href="{{ $ctrl.uiActions.manageAutorenew.href }}"
+                                data-external>
+                            </oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{::'common_tile_subscription_contact' | translate}}">
+                        <oui-tile-description>
+                            <ul class="list-unstyled">
```

Comment:
`oui-tile-definition` is already a dl > dt + dd
Take a look: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-tile/src/definition/tile-definition.html :)

Reply:
@AxelPeter Using multiple oui-tile-description makes it appear inline so have used a list here.

Reply:
@FredericEspiau i tried with `dl` and `dd` but there will be no `dt` inside....it will be only `dl` and `dd`.
Felt a bit odd so used `ul` and `li` instead

Reply:
`oui-tile-description` is transclude slot, not a component
https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-tile/src/definition/tile-definition.component.js

Reply:
It was just a question 😅, I felt like the `span` could have been replaced with a `dt` and the `small` with a `dl`, but I couldn't know 😃

Reply:
@AxelPeter Then `ul` is the best way to go forward in description to get list of elements? 
Or can use empty div here that can be inline block here.